index.css

```diff
@@ -456,7 +658,9 @@ li {
     display: flex;
     flex-direction: column;
     height: 100%;
-    gap: 20px;
+    width: 100%;
+    padding: 0;
+    gap: 30px;
   }
 
   .contact-wrap form {
```

Comment:
- [ ] In order to produce the exact the design as we have it on the Figma design, kindly increase the `width` of the form.

###  Your Design
![your-form](https://user-images.githubusercontent.com/26597432/220650639-cebffdf8-f2b0-4f91-be26-874b62b1ec01.png)

### Figma Design
![temp-form](https://user-images.githubusercontent.com/26597432/220650730-3a9dbd16-7ebe-4444-873f-4b38bb267a76.png)

Reply:
Width adjusted